README.md

```diff
@@ -100,6 +100,7 @@ See src/nodejs/ for the following:
   * Automatically select nodes
   * Get nodes by configurable sort factor
   * GetNodesByX
+    * Be careful, this can produce a lot of node traffic. It first pings each node in the list generated or provided to make sure they are up and within operating parameters and then calls the respective method requested. See [neo-rpc](#neo-rpc) for examples.
```

Comment:
Should there be a confirmation prompt before executing this call with this message? Many people won't read this

Reply:
Excellent idea! Thanks